tabfilter.py

```diff
@@ -16,25 +16,35 @@ class TabFilterCommand(sublime_plugin.WindowCommand):
 
 	def run(self):
 		"""Shows a quick panel to filter and select tabs from the active window"""
-		
+
 		tabs = []
 		self.window = sublime.active_window()
 		self.views = []
 		self.prefix = ""
 		self.settings = sublime.load_settings("tabfilter.sublime-settings")
 
+		idx = 0
 		for view in self.window.views():
 			self.views.append(view)
+			if self.window.active_sheet().view().id() == view.id():
+				""" save index for later usage"""
```

Comment:
For consistency, do you mind changing this to a hash style comment, i.e. `# save index for later usage` - I try to stick to pep8 and try to stick multi-line strings as documentation blocks for method signatures (or for legit multi-line strings).